1-js/02-first-steps/14-switch/article.md

```diff
@@ -1,22 +1,22 @@
-# The "switch" statement
+# A instrução "switch"
 
-A `switch` statement can replace multiple `if` checks.
+Uma instrução `switch` pode substituir muitas comparações `if` (se).
 
-It gives a more descriptive way to compare a value with multiple variants.
+Ela fornece uma forma mais descritiva para comparar um valor a múltiplas variantes.
```

Comment:
Sugestão de alteração:

- Alterar `forma mais` por `maneira mais`

<img width="655" alt="image" src="https://user-images.githubusercontent.com/35238611/195748703-4a4bdde2-eedf-4f2e-8a1c-c7b6af389f10.png">

> Motivo de cacofonia
> https://www.significados.com.br/cacofonia/

Reply:
Trying to be formal. Another suggestion?

Reply:
Embora a sugestão fornecida, acredito que não impacte tanto na interpretação do leitor ao paragrafo. Sendo assim, entendo que podemos manter a expressão.

Reply:
@jonnathan-ls , Hope it is all right now. Thanks.